Core/clim-core/frames.lisp

```diff
@@ -623,17 +622,14 @@ documentation produced by presentations.")
 
 (defmethod make-pane-1 :around (fm (frame standard-application-frame) type
                                 &rest args
-                                &key (input-buffer nil input-buffer-p)
-                                     name
-                                &allow-other-keys)
-  (declare (ignore name input-buffer))
-  "Default input-buffer to the frame event queue."
-  (let ((pane (if input-buffer-p
-                  (call-next-method)
-                  (apply #'call-next-method fm frame type
-                         :input-buffer (frame-event-queue frame)
-                         args))))
-    pane))
+                                &key (event-queue nil evq-p)
+                                  &allow-other-keys)
```

Comment:
Did the indentation get messed up?

Reply:
that's how slime indents it. I've put it in the same line.